apps/web/screens/ApiCatalogue/ApiCatalogue.tsx

```diff
@@ -412,6 +400,11 @@ const ApiCatalogue: Screen<ApiCatalogueProps> = ({
               {data?.getApiCatalogue?.services.length > 0 && (
                 <GridContainer>
                   <ServiceList
+                    baseUrl={
+                      activeLocale === 'en'
+                        ? './webservices/'
+                        : './vefthjonustur/'
+                    }
```

Comment:
If you are resolving to the url `/throun/vefthjonustur` you could do it like this:
```ts
const { linkResolver } = useLinkResolver() // at the top of this cmp
...
<ServiceList
  baseUrl={linkResolver('webservicespage').as} // linkResolver ensures this link is of right locale
```